org.kde.kaffeine.json

```
@@ -0,0 +1,263 @@
+{
+  "app-id": "org.kde.kaffeine",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15-21.08",
+  "sdk": "org.kde.Sdk",
+  "add-extensions": {
+    "org.freedesktop.Platform.ffmpeg-full": {
+        "directory": "lib/ffmpeg",
+        "version": "21.08",
+        "add-ld-path": ".",
+        "no-autodownload": true,
+        "autodelete": false
+    }
+  },
+  "command": "kaffeine",
```

```suggestion
  "command": "kaffeine",
  "rename-icon": "kaffeine",
```

This could help, but I am not sure.

---

I think it would.
Toward the end of the build I get:
```
WARNING: Icon not matching app id in /var/home/hub/source/flathub/org.kde.kaffeine/repo/export/share/applications/org.kde.kaffeine.desktop: kaffeine
WARNING: Icon referenced in desktop file but not exported: kaffeine
```

---

The installed destkop file has `Icon=kaffeine` which `rename-icon` will fix.